tests/components/ct-vue-vite/src/notation-jsx.spec.tsx

```diff
@@ -151,3 +152,17 @@ test('render app and navigate to a page', async ({ page, mount }) => {
   await expect(component.getByRole('main')).toHaveText('Dashboard');
   await expect(page).toHaveURL('/dashboard');
 });
+
+test('render global store state', async ({ mount }) => {
```

Comment:
Same as in #18765, let's start slicing tests by feature rather than putting everything in one test!